### PR TITLE
`az appservice kube` : allow passing custom location by name, add ASEv3 SKUs, bug fix

### PR DESCRIPTION
**Pull request by StrawnSC**
---

- Allows `--custom-location` parameter to be passed by name instead of resource ID if in the RG as the app/plan 
- Adds ASEv3 SKUs to the `az appservice plan create`
- Fixes bugs caused by appservice kube environment RG's being returned as null 

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [X] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [X] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [X] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by jiasli** [comment]
appservice-kube

File: src/appservice-kube/HISTORY.rst [hunk 1]
```diff
@@ -11,3 +11,9 @@ Release History
 0.1.1
 ++++++
 * Fix ssl binding for web apps in kubernetes environments
+
+0.1.2
+++++++
+* Allow passing custom locations by name if in the same resource group as the app/plan
+* Add App Service Environment V3 SKUs to 'az appservice plan create' help text
+* Fix bug causing App Service Kube Environment "resourceGroup" value to be null in command responses
```
Comment (zhoxing-ms) [inline src/appservice-kube/HISTORY.rst:19]: Please write the latest history on the top

**Comment by zhoxing-ms** [comment]
Could you add some tests for this new change?

**Comment by panchagnula** [comment]
> Could you add some tests for this new change?

@zhoxing-ms since this is still in extension & experimental, we plan to write tests when we start porting this to core-cli. Hope that works?

File: src/appservice-kube/azext_appservice_kube/_params.py [hunk 2]
```diff
@@ -93,7 +93,7 @@ def load_arguments(self, _):
                    completer=get_resource_name_completion_list('Microsoft.Web/serverFarms'),
                    help="name or resource id of the function app service plan. Use 'appservice plan create' to get one")
         c.argument('new_app_name', options_list=['--name', '-n'], help='name of the new function app')
-        c.argument('custom_location', help="Name or ID of the custom location")
+        c.argument('custom_location', help="Name or ID of the custom location. Use an ID for a custom location in a different resource group from the app")
```
Comment (panchagnula) [inline src/appservice-kube/azext_appservice_kube/_params.py:96]: can we update help.py with an example for using custom-location ID in a different RG so the usage understood?

**Review by panchagnula — APPROVED** [review]
LGTM - the one comment regarding add an example about usage in help.py

**Comment by StrawnSC** [comment]
Not sure why the CI tests are failing. They seem unrelated to my change

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**